css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,15 @@
+button:hover {
+  background-color: green;
+}
+
+.table-headings {
+  background-color: black;
+}
+
+.heading {
+  color: white;
+}
+
+.has-background {
```

Comment:
This will totally work, but please redo this to make use of a pseudo-class rather than a regular class 👍